data/saas/dataset/stripe_dataset.yml

```diff
@@ -634,6 +661,10 @@ dataset:
                     data_categories: [user.contact.address.country]
                     fidesops_meta:
                       data_type: string
+                      masking_strategy_override:
+                        strategy: string_rewrite
+                        configuration:
+                          rewrite_value: US
```

Comment:
What's the reasoning for using a rewrite value of `US`? I see in other places we flag the country as read-only so we don't modify anything. Can we do that here?